cli/data.go

```diff
@@ -455,7 +455,7 @@ func (c *viamClient) tabularData(dst string, filter *datapb.Filter) error {
 	}
 	w := bufio.NewWriter(dataFile)
 
-	fmt.Fprintf(c.c.App.Writer, "Downloading..") // no newline
+	fmt.Fprintf(c.c.App.ErrWriter, "Downloading..") // no newline
```

Comment:
Justification: Most cli tools that I know of put progress bars / info on stderr instead of stdout.

Reply:
So, I agree it's bad to have a changing "progress bar" type of output going to a script. However, I'm not sure most cli programs actually write those to STDERR either. STDOUT is more about "main data channel" and stderr is "debug and other sideband stuff." The programs that put progress on STDERR (e.g. wget, curl, and pv) are typically doing so because the actual data they're fetching is what's going through STDOUT. But where bulk data output isn't expected, STDOUT seems to be the place. Apt, git, and brew all write progress to stdout, for example.

Reply:
Mhmmm. Okay, I think you are right about this one. reverting and keeping progress bars on stdout. Thank you for checking other tools that I had forgot to check

Reply:
So... if we DON'T implement a "script" mode of some kind, then I'm kinda torn about if it should move to stderr or not. It's "sideband" data from a scripting perspective (and your instinct to move it for that reason feels correct), but it's something a human user would pretty much always want to see. But... if we do thing eventually we'll do script-mode or bits of it, then leaving it on stdout makes sense, as that's where it probably belongs for human-mode anyway (and it's not a line likely to be confused for the output a script would be looking for, so zero-cost to ignore for a script author.)

Reply:
> (and it's not a line likely to be confused for the output a script would be looking for, so zero-cost to ignore for a script author.)

Agreed. I think it is probably better to keep it on stdout for humans permanently, and once we add a script mode, we can decide what we want to do with progress info.

(swapping back and forth between stdout and stderr in the future is a breaking change that is probably worse than just having users ignore the progress info for now imo)

Reply:
I suspect we will/should add a script mode. We want people to be able to interact with their robots programmatically and the cli is a great way to do simple things.